Small changes

zamerman opened the pull request:
Just getting in on this a little early both to familiarize myself with github and refresh myself on html.

ForgivenNin commented:
Sweet dude! 
Thanks for adding all this! I know that I added a couple projects to this page, but let's totally put in some more HTML stuff. Hopefully those websites I added are of use.

Also, could you explain the spacing? I find it really odd for the way programmers space out all their brackets and stuff.

zamerman commented:
Nice to hear from you!
Sure thing, although I just started another programming project on my own so I might not contribute much till maybe next week.

As for the spacing, spacing and indentation is more for the sake of working with other programmers than for any real reason within the program structure. Of course, there are languages like python where indentation is part of the syntax, but that’s another story. Spacing and indentation makes the code easier to read, combine that with comments and if another programmer jumps in on the project midway they can get up to speed a lot faster. It also helps when your going back over the code and making changes since it makes it easier for you to read as well.

Sent from Mail for Windows 10

From: ForgivenNin
Sent: Saturday, January 26, 2019 8:52 PM
To: ForgivenNin/Playground-Experimental-HTML
Cc: Zachary Amerman; Author
Subject: Re: [ForgivenNin/Playground-Experimental-HTML] Small changes (#1)

Sweet dude!
Thanks for adding all this! I know that I added a couple projects to this page, but let's totally put in some more HTML stuff. Hopefully those websites I added are of use.
Also, could you explain the spacing? I find it really odd for the way programmers space out all their brackets and stuff.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
